### PR TITLE
use webxdc types package

### PR DESCRIPTION
**Pull request by Simon-Laux**
- [ ] needs to be tested if it still works

note that there are some ts errors with solids stuff that were there before already (on master).

### CODE REVIEW

**Comment by WofWca** [comment]
Did you mean to simply rename the old file instead of deleting it?

File: sim/create.ts [hunk 1]
```diff
@@ -85,112 +85,116 @@ export function createWebXdc(
...
+            "data:application/octet-stream;base64," + base64Content
+          );
+          element.setAttribute("download", content.file.name);
+          document.body.appendChild(element);
+          element.click();
+          document.body.removeChild(element);
+        }
+      } else {
+        alert(msg);
+      }
```
Comment (WofWca) [inline sim/create.ts:169]: I'd put formatting changes and semantic changes in different commits for easier reviewing. The diff view on GitHub looks really huge, whereas it's actually all just whitespace.

**Review by WofWca — APPROVED** [review]
Works for me, why wouldn't it? Both `npm run build` and `npm run dev`

**Comment by adbenitez** [comment]
the types package is now at `@webxdc/types`

**Comment by adbenitez** [comment]
superseded by #61

**Pull request closed without merging**